docs/templates/solc_0.8/permit/Permit.md

```diff
@@ -0,0 +1,92 @@
+---
+description: Permit
+---
+
+# Permit
+
+## Introduction
+
+*The purpose of this contract is to allow transfer from an owner ERC-20 extended to a spender one*
+
+## Model
+
+*This contract is dealing with sand (an ERC20 extended).*  
+*[ERC20](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/) is the classic standard to represent fungible token.*  
+*ERC20 extended only had the ability to burn (destroy) tokens.*  
+
+## Process
+
+### Step 1
+
+*we ensure that deadline authorization has not expired*  
```

Comment:
What is `deadline authorization` ? explain to the lector

Reply:
ok